src/covid_model_seiir_pipeline/cli.py

```diff
@@ -24,7 +24,7 @@ def seiir():
               help="Either a location set version id used to pull a list of"
                    "locations to run, or a full path to a file describing"
                    "the location set.")
-@cli_tools.add_output_options(default_output_root=DEFAULT_SEIIR_OUTPUT_ROOT)
+@cli_tools.add_output_options(paths.SEIR_FIT_OUTPUTS)
```

Comment:
`DEFAULT_SEIIR_OUTPUT_ROOT` can be safely deleted now, right?

Reply:
Oh, yes